src/native/procs.h

```diff
@@ -126,3 +126,7 @@
 	PFNGLGETSTRINGPROC glGetString;
 	PFNGLGETERRORPROC glGetError;
 	PFNGLDRAWBUFFERSEXTPROC glDrawBuffersEXT;
+        PFNGLGENVERTEXARRAYSOESPROC glGenVertexArraysOES;
```

Comment:
Nit pick: can you fix the indentation for these lines here to be two fewer spaces?

Reply:
Looks like this file has actual tab characters instead of spaces (my editor added spaces). Should I update the whole file to spaces?

Reply:
Right - I didn't notice. Let's keep it all tabs, then.

Reply:
Sounds good. Updated here and in procs.cc